packages/stark-ui/src/modules/message-pane/components/message-pane.component.ts

```diff
@@ -62,47 +62,47 @@ export class StarkMessagePaneComponent extends AbstractStarkUiComponent implemen
 	/**
 	 * Messages tab currently selected
 	 */
-	public currentNavItem: StarkMessagePaneNavItem;
+	public currentNavItem: StarkMessagePaneNavItem = "";
 
 	/**
 	 * Total number of messages currently shown
 	 */
-	public totalMessages: number;
+	public totalMessages = 0;
 
 	/**
 	 * Maximum level of the messages currently shown (error, warning or info).
 	 */
-	public maxLevel: StarkMessagePaneNavItem;
+	public maxLevel: StarkMessagePaneNavItem = "infos";
```

Comment:
I'm not sure if this change will cause a regression... could you double check this?

Reply:
This gets it value through `this.getMaxLevel(msgCollection)` which returns "infos" by default. So I think this should be fine.